official/enfuse.lua

```diff
@@ -149,6 +150,14 @@ dt.register_lib(
             cnt = cnt + 1
             f:write(i.path.."/"..i.filename.."\n")
             target_dir = i.path
+          elseif i.is_raw and darktable_cli_installed then
+            local tmp_exported = os.tmpname()..".tif"
+            os.execute("darktable-cli \""..i.path.."/"..i.filename.."\" \""..tmp_exported.."\"")
+            dt.print("Raw file "..i.filename.." automatically converted to tiff")
```

Comment:
If you would use "string.format(" here instead of joining the strings with ".." it would make it easier to translate.  And then using "_()" would it even make it possible to translate the string. (Yes I know that the rest of the script is not translatable. But we have to start somewhere.)

https://github.com/darktable-org/lua-scripts/blob/3a5dfbc842c27596fe7cd7b1fa58030380df9eb5/contrib/image_stack.lua#L253

Reply:
Not a problem at all, and it's a good thing to start from there! Shall I fix the string (and I'd also move the `print` before the conversion command) and reissue the pull request?

Reply:
Yes, please.